docs/src/main/tut/typeclasses/typeclasses.md

```diff
@@ -221,6 +221,8 @@ val result = Monoid[Int].combine(sumLeft, sumRight)
 Cats provides laws for type classes via the `kernel-laws` and `laws` modules which makes law checking
 type class instances easy.
 
+You can find out more about law testing [here](typeclasses/lawtesting.html).
```

Comment:
Sorry I should've caught this earlier. This link probably has an extra typeclasses in path?

Reply:
@kailuowang The path is good. I've tested it.
Without the `typeclasses` prefix the URL formed is: /cats/lawtesting, but the original .md is placed in typeclasses/lawtesting.md.

Reply:
oh cool, thanks for testing it.